core/server/common/src/main/java/alluxio/cli/extensions/command/LsCommand.java

```diff
@@ -38,8 +40,8 @@
    *
    * @param extensionsDir the path to the directory containing the extensions
    */
-  public LsCommand(String extensionsDir) {
-    mExtensionsDir = extensionsDir;
+  public LsCommand() {
+    mExtensionsDir = ServerConfiguration.get(PropertyKey.EXTENSIONS_DIR);
```

Comment:
Because we can just call `ServerConfiguration.get` why don't we make an empty constructor here and just make the `ServerConfiguration` call when the command is actually executing. I think it just simplifies the code since the `mExtensionsDir` isn't used anywhere

Reply:
done